packages/backend/src/peripherals/database/StateUpdateRepository.ts

```diff
@@ -291,3 +302,41 @@ function toStateUpdatePriceRecord(
     price: record.price,
   }
 }
+
+function encodeState(state: State): ToJSON<State> {
```

Comment:
First thought was that this fn should be in encoding package, but clearly it should not. I would change the name encode/decode to serialize/deserialize or smth

Reply:
I renamed to `stateToJson` and `stateFromJson`